docs/src/theming.md

```diff
@@ -11,4 +18,4 @@ Attributes(
     linewidth = 3
 )
 ```
-To start Makie always with this theme!
+then Makie will with this theme.
```

Comment:
Maybe reword to something like:

```then Makie will start with this theme.```

?

Reply:
yeah, I don't know how that appeared 😆